src/Symfony/Component/Validator/CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ CHANGELOG
    ```
  * Add support for ratio checks for SVG files to the `Image` constraint
  * Add the `Slug` constraint
+ * Add support for enums in `Choice` constraint
```

Comment:
Enums are already supported

This PR add  supports for using a class-string<UnitEnum> to specify the list of possible values ( integer or string )